### PR TITLE
Fix #54

### LINKED ISSUE #54: animation speed broken in :host(.animate)
I am trying to change the animation speed. When adjusting the seconds, there is no effect:
1.0.4 version

``` css
   :host(.animate) {
      /* transition */
      transition: height 3s ease-in;
    }
```

### PR DESCRIPTION
**Pull request by blasten**
Fixes #54

### CODE REVIEW

File: paper-toolbar.html [hunk 1]
```diff
@@ -87,6 +87,7 @@
 `--paper-toolbar`            | Mixin applied to the toolbar | `{}`
 `--paper-toolbar-medium`     | Mixin applied to medium height toolbar | `{}`
 `--paper-toolbar-tall`       | Mixin applied to tall height toolbar | `{}`
+`--paper-toolbar-transition` | Transition applied to the `.animate` class | `height 0.18s ease-in`
```
Comment (bicknellr) [inline paper-toolbar.html:94]: Is something else expected to add this class? If not, this might be a good time to document it.

Comment (blasten) [inline paper-toolbar.html:94]: will fix

**Comment by bicknellr** [comment]
ping

File: paper-toolbar.html (hunk 1 above)
Comment (addyosmani) [inline paper-toolbar.html:94]: Friendly ping @blasten :cake:

**Comment by blasten** [comment]
@bicknellr @addyosmani There you go.!

File: paper-toolbar.html [hunk 2]
```diff
@@ -100,7 +104,7 @@
 <dom-module id="paper-toolbar">
   <template>
     <style>
-      :host {
+      :host {`
```
Comment (bicknellr) [inline paper-toolbar.html]: stray ```

Comment (blasten) [inline paper-toolbar.html]: O.o  Fixed! @bicknellr

**Comment by bicknellr** [comment]
LGTM

**Pull request merged**